src/core/Akka/Event/LoggingBus.cs

```diff
@@ -108,28 +107,13 @@ internal void StartDefaultLoggers(ActorSystemImpl system)
                     continue;
                 }
 
-                if (asyncStart)
```

Comment:
This is not relevant anymore, ActorSystem startup will not throw on logger initialization timeout.

Reply:
When did we make that change? Is there a PR you can link to here?